src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -11,6 +11,9 @@
 
 module Relation.Nullary.Decidable.Core where
 
+-- this can go through `Data.Maybe.Base` once the deprecation of `decToMaybe`
+-- is fully done.
```

Comment:
```suggestion
-- decToMaybe was deprecated in v2.1
-- this can go through `Data.Maybe.Base` once the deprecation of `decToMaybe`
-- is fully done.
```
Let's reuse the magic string to make greping for things that need to be removed easier.